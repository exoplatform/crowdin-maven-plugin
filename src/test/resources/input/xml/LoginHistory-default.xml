<?xmfl version='1.0' encoding='UTF-8'?>
<!--

    Copyright (C) 2009 eXo Platform SAS.
    
    This is free software; you can redistribute it and/or modify it
    under the terms of the GNU Lesser General Public License as
    published by the Free Software Foundation; either version 2.1 of
    the License, or (at your option) any later version.
    
    This software is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
    Lesser General Public License for more details.
    
    You should have received a copy of the GNU Lesser General Public
    License along with this software; if not, write to the Free
    Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
    02110-1301 USA, or see the FSF site: http://www.fsf.org.

-->

<messagebundle>
  <msg name="loginHistory.title">
    Login History
  </msg>
  <msg name="loginHistory.user">
    User
  </msg>
  <msg name="loginHistory.find">
    Find
  </msg>
  <msg name="loginHistory.latest">
    Reset
  </msg>
  <msg name="loginHistory.lastLogin">
    Last login
  </msg>
  <msg name="loginHistory.beforeThat">
    Before that
  </msg>
  <msg name="loginHistory.more">
    More
  </msg>
  <msg name="loginHistory.stats">
    stats
  </msg>
  <msg name="loginHistory.history">
    history
  </msg>
  <msg name="loginHistory.global">
    Global
  </msg>
  <msg name="loginHistory.week">
    Week
  </msg>
  <msg name="loginHistory.lastLoginsView">
    Last logins
  </msg>
  <msg name="loginHistory.historyView">
    History
  </msg>
  <msg name="loginHistory.statisticsView">
    Statistics
  </msg>
  <msg name="loginHistory.averagePerDay">
    Average logins / day
  </msg>
  <msg name="loginHistory.averagePerWeek">
    Average logins / week
  </msg>
  <msg name="loginHistory.averagePerMonth">
    Average logins / month
  </msg>
  <msg name="loginHistory.today">
    Today
  </msg>
  <msg name="loginHistory.earlierThisWeek">
    Earlier this week
  </msg>
  <msg name="loginHistory.earlierThisMonth">
    Earlier this month
  </msg>
  <msg name="loginHistory.earlier">
    Earlier
  </msg>
  <msg name="loginHistory.allUsers">
    All users
  </msg>
  <msg name="loginHistory.unknow">
    unknow
  </msg>
  <msg name="loginHistory.next">
    next
  </msg>
  <msg name="loginHistory.ago">
    ago
  </msg>
</messagebundle>

