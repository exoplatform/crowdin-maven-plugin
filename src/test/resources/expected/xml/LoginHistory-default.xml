<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<messagebundle>
  <msg name="loginHistory.title">Login History</msg>
  <msg name="loginHistory.user">User</msg>
  <msg name="loginHistory.find">Find</msg>
  <msg name="loginHistory.latest">Reset</msg>
  <msg name="loginHistory.lastLogin">Last login</msg>
  <msg name="loginHistory.beforeThat">Before that</msg>
  <msg name="loginHistory.more">More</msg>
  <msg name="loginHistory.stats">stats</msg>
  <msg name="loginHistory.history">history</msg>
  <msg name="loginHistory.global">Global</msg>
  <msg name="loginHistory.week">Week</msg>
  <msg name="loginHistory.lastLoginsView">Last logins</msg>
  <msg name="loginHistory.historyView">History</msg>
  <msg name="loginHistory.statisticsView">Statistics</msg>
  <msg name="loginHistory.averagePerDay">Average logins / day</msg>
  <msg name="loginHistory.averagePerWeek">Average logins / week</msg>
  <msg name="loginHistory.averagePerMonth">Average logins / month</msg>
  <msg name="loginHistory.today">Today</msg>
  <msg name="loginHistory.earlierThisWeek">Earlier this week</msg>
  <msg name="loginHistory.earlierThisMonth">Earlier this month</msg>
  <msg name="loginHistory.earlier">Earlier</msg>
  <msg name="loginHistory.allUsers">All users</msg>
  <msg name="loginHistory.unknow">unknow</msg>
  <msg name="loginHistory.next">next</msg>
  <msg name="loginHistory.ago">ago</msg>
</messagebundle>
